Include/codecs.h

```diff
@@ -27,6 +27,16 @@ PyAPI_FUNC(int) PyCodec_Register(
        PyObject *search_function
        );
 
+/* Unregister a codec search function.
+
+   Unregister a codec search function and clear the registry's cache.
+   If the search function is not registered, do nothing.
```

Comment:
Would you mind to document what are the possible return values?

I understood:

"Return 0 on success. Raise an exception and return -1 on error."

Reply:
updated.